content/lua/concepts/coroutines/terms/create/create.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'create()'
+Description: 'Returns a new coroutine with a function as an argument that will run within the new coroutine.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Lua'
```

Comment:
```suggestion
  - 'Methods'
```
We generally do not create tags for things that are already topics in Docs, as they would be redundant.